subprojects/core-api/src/main/java/org/gradle/api/file/CopyProcessingSpec.java

```diff
@@ -86,6 +88,8 @@ public interface CopyProcessingSpec extends ContentFilterable {
      * Returns the Unix permissions to use for the target files. {@code null} means that existing
      * permissions are preserved. It is dependent on the copy action implementation whether these permissions
      * will actually be applied.
+     * <p>
+     * Use {@link #getFilePermissions()} instead.
```

Comment:
❓ This method is not deprecated, yet we say "Use X instead". Feels incorrect.

Reply:
Good catch. It's a leftover from having deprecations added in this PR but then pulling them out in a separate PR.